documentation/src/docs/getting-started.md

```diff
@@ -73,14 +73,13 @@ In order to build a new version of Flink for a different Scala version, please r
 
 ## Setting up your IDE
 
-Connector project uses [Project Lombok](https://projectlombok.org/), so we should ensure that we have our IDE setup with the required plugins. (**IntelliJ is recommended**).
+IntelliJ is recommended for project connector.
 
 To import the source into IntelliJ:
 
 1. Import the project directory into IntelliJ IDE. It will automatically detect the gradle project and import things correctly.
 2. Enable `Annotation Processing` by going to `Build, Execution, Deployment` -> `Compiler` > `Annotation Processors` and checking `Enable annotation processing`.

```

Comment:
I think we don't need annotation processing step as well.

Reply:
fixed in [7d9a421](https://github.com/pravega/flink-connectors/pull/613/commits/7d9a421f16b3b1536a24d1312853be87f500c9f2)